std-bits/table/src/main/java/org/enso/table/data/column/builder/BigDecimalBuilder.java

```diff
@@ -6,16 +6,21 @@
 import org.enso.table.data.column.storage.type.BigDecimalType;
 import org.enso.table.data.column.storage.type.StorageType;
 import org.enso.table.error.ValueTypeMismatchException;
+import org.enso.table.problems.ProblemAggregator;
 
 /** A builder for BigDecimal columns. */
 public class BigDecimalBuilder extends TypedBuilderImpl<BigDecimal> {
+  // The problem aggregator is only used so that when we are retyping, we can pass it on.
+  private final ProblemAggregator problemAggregator;
```

Comment:
Where is that usage? I may be missing something but I don't see it.

Reply:
Oops. I initially did not have this parameter, then I needed it, but it looks like I removed what was using it.